scripts/run_goreleaser.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+[[ -z ${DEBUG:-} ]] || set -o xtrace
+
+OS="$(uname -s | tr '[:upper:]' '[:lower:]')"
+DIR="$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && cd .. && pwd -P)"
+GITCONFIG_VOLUME=${GIT_CONFIG:-"${HOME}/.gitconfig"}
+GCLOUD_CONFIG_DIR="${CLOUDSDK_CONFIG:-${HOME}/.config/gcloud}"
+SSH_CONFIG_DIR_VOLUME="${HOME}/.ssh"
+SSH_AUTH_SOCK_ENV_VAR="/ssh-agent"
+SSH_AUTH_SOCK_VOLUME="${SSH_AUTH_SOCK}:/ssh-agent"
+
+if [[ "${BUILDKITE:-}" == "true" ]]; then
+    GITCONFIG_VOLUME="/var/lib/buildkite-agent/.gitconfig"
+    SSH_CONFIG_DIR_VOLUME="/var/lib/buildkite-agent/.ssh"
+fi
+
+# On Mac for Desktop we need to use these hardcoded values when forwarding SSH agent socket
+# Docker would 'magically' map whatever is set in $SSH_AUTH_SOCK for the user to the container ns
+if [[ "${OS}" == "darwin" ]]; then
+    SSH_AUTH_SOCK_VOLUME="/run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock"
```

Comment:
is /run/host-services expected to exist as a path on a macos host? I wasn't able to find it on my mac (running sonoma)

Reply:
I don't recall the context here since its been a very long time, but I wouldn't be surprised if we used this for some weird docker issue - since releases will happen via CI, we can 🔪 this.